src/bindings/python/src/pyopenvino/pyopenvino.cpp

```diff
@@ -98,21 +98,37 @@ PYBIND11_MODULE(_pyopenvino, m) {
 
     m.def("get_version", &get_version);
     m.def("get_batch", &ov::get_batch);
-    m.def("set_batch", &ov::set_batch);
+    m.def(
+        "get_batch",
+        [](const py::object& ie_api_model) {
+            const auto model = ie_api_model.attr("_Model__model").cast<std::shared_ptr<ov::Model>>();
+            return ov::get_batch(model);
+        },
+        py::arg("model"));
+    m.def(
+        "set_batch",
```

Comment:
Since that's a new binding, do we have tests for it?

Reply:
I didn't really add a new bindings. In this case we had
```py
m.def("set_batch", &ov::set_batch);
```
which implicitly took `pyopenvino.Model` and `pybind` was able to automatically cast it to ov::Model. As we removed inheritance, `pybind` no longer is able to cast it. 
https://github.com/openvinotoolkit/openvino/blob/ea72f3047cb2fca24a89bed7c6069d4abb960a99/src/bindings/python/tests/test_runtime/test_model.py#L337